wfe/web-front-end.go

```diff
@@ -170,14 +171,14 @@ func (wfe *WebFrontEndImpl) HandleFunc(mux *http.ServeMux, pattern string, h fun
 		}
 		response.Header().Set("Access-Control-Allow-Origin", "*")
 
-		switch request.Method {
-		case "HEAD":
+		// Return a bodyless response to HEAD for any resource that allows GET.
+		if _, ok := methodsOK["GET"]; ok && request.Method == "HEAD" {
```

Comment:
This seems to breaks stuff: we no longer handle the case where `request.Method == "HEAD" && !methodsOK["GET"]` so we erroneously send a body.